docs/usage/configuration-options.md

```diff
@@ -1767,6 +1767,11 @@ Normally when you set `rebaseWhen=auto` Renovate rebases any branch that's behin
 This behavior is no longer guaranteed when you enable `platformAutomerge` because the platform might automerge a branch which is not up-to-date.
 For example, GitHub might automerge a Renovate branch even if it's behind the base branch at the time.
 
+## platformCommit
+
+Commits will be performed by the platform API rather than using `git` directly.
+Supports GitHub-only.
```

Comment:
```suggestion
Supports GitHub-only.

The primary reason to use this option is because commits will then be signed automatically, including if authenticating as an app.
```